lib/templates/src/config.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
I put the config file in the root of templates/src

Reply:
Cool! Thank you.